addon/mixins/editable.js

```diff
@@ -40,11 +39,21 @@ export default Mixin.create({
       }
 
       // sets the global editing status to let the table know of any on going editing
-      this.manager.set('editStatus', {
+      let editableStatus = {
         key: this.column.key,
         status: 'editing',
-        item: this.item
-      });
+        item: this.item,
+        id: this.element.id
+      }
+
+      if(!elementEditStatus) {
+        let newStatus = [...editStatus, editableStatus]
+        this.manager.set('editStatus', newStatus)
+      } else {
+        let index = editStatus.indexOf(elementEditStatus)
+        let newStatus = [...editStatus].replace(index, 1, [editableStatus])
```

Comment:
why not directly manipulate the array `editStatus` ?

Reply:
I tried it at first, didn't work
when doing `let newStatus = editStatus.replace(index, 1, [editableStatus])` ember doesn't detect the changes, unless it's a new copy, might be immutable

Reply:
And what about have a hash instead of an array for `editStatus` with the column key has key ?

Reply:
That could be better, I will try it today